bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/auxfile/AttachmentTableModel.java

```diff
@@ -66,24 +66,16 @@ public int getColumnCount() {
         return Col.values().length;
     }
 
-    public Object getValueAt(int row, int col) {
-        AuxFile file = fileList.get(row);
+    public Object getValueAt(final int row, final int col) {
+        final AuxFile file = fileList.get(row);
         return Col.values()[col].getValue(file);
     }
 
-    public String getColumnName(int col) {
+    public String getColumnName(final int col) {
         return Col.values()[col].displayValue();
     }
 
-    public Class getColumnClass(int col) {
+    public Class getColumnClass(final int col) {
         return Col.values()[col].getColumnClass();
     }
-
-    @Override public boolean isCellEditable(int rowIndex, int columnIndex) {
-        return false; // columnIndex == Col.CHECKED.ordinal();
```

Comment:
Always returning `false` is the default implementation, so there was no need to override now that the old code was commented out.